packages/form-js-playground/test/spec/Playground.spec.js

```diff
@@ -48,6 +48,20 @@ describe('playground', function() {
       approvedBy: 'John Doe',
       mailto: ['regional-manager', 'approver'],
       product: 'camunda-cloud',
+      dynamicDataTestData: [
```

Comment:
I like that the playground test data mostly follows a real-world use case here. Maybe we can come up with something less generic than `dynamicTestData`? Something about currencies (Euro, Dollar, Pound, ...)  

I'm also not super creative though 😅